.github/workflows/publish.yml

```diff
@@ -45,3 +45,19 @@ jobs:
             LICENSE
             dist/*sqlfmt*.whl
             dist/*sqlfmt*.tar.gz
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
```

Comment:
It seems I've made a mistake and this should have been
```yaml
registry: ghcr.io
```

Reply:
yep, fixed in #368 . And I manually pushed to ghcr for `latest` and `v0.15.1`